packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -689,8 +690,8 @@ export const makeLegacySendBatchFn =
  * local - Did this client send the op?
  * savedOp - Is this op being replayed after being serialized (having been sequenced previously)
  * batchStartCsn - The clientSequenceNumber given on submit to the start of this batch
```

Comment:
This property comment seems to be obsolete (there is no matching property). Should it be removed?

Reply:
Good eyes!